airbyte-webapp/src/components/CreateConnectionContent/CreateConnectionContent.tsx

```diff
@@ -76,14 +63,12 @@ const CreateConnectionContent: React.FC<CreateConnectionContentProps> = ({
       sourceCatalogId: catalogId,
     });
 
-    return {
-      onSubmitComplete: () => {
-        afterSubmitConnection?.(connection);
-      },
-    };
+    // This is a bit of a hack. I wanted to get away from our previous way of handling this
+    // but our form dirty tracking needs a second to catch up w/ the form reset + submit.
+    setTimeout(() => push(`../../connections/${createdConnection.connectionId}`), 1000);
```

Comment:
Does it make sense to make this part of the after-submit _always_?  When you edit a connection on the replication view, it often (always?) triggers a reset.  It could make sense that submitting the form takes you to `/connection/:id` where you can see the status of that job.